task/runner.go

```diff
@@ -258,4 +261,31 @@ func errorInfo(err error) *data.ErrorInfo {
 	return &data.ErrorInfo{Message: err.Error(), Unretriable: IsUnretriable(err)}
 }
 
+func humanizeError(err error) error {
+	if err == nil {
+		return nil
+	}
+
+	errMsg := strings.ToLower(err.Error())
+
+	if strings.Contains(errMsg, "unexpected eof") {
+		return errors.New("file download failed")
+	} else if strings.Contains(errMsg, "multipartUpload: upload multipart failed") {
+		return errors.New("internal error saving file to storage")
+	} else if strings.Contains(errMsg, "mp4io: parse error") {
+		return UnretriableError{errors.New("file format unsupported, must be MP4")}
+	}
```

Comment:
Could you add another check here for a new error we have?
```suggestion
	}
	
	isTimeout := strings.Contains(errMsg, "context deadline exceeded") ||
	    strings.Contains(errMsg, "context canceled") ||
	    strings.Contains(errMsg, "context deadline exceeded")
	 if isTimeout {
	     return errors.New("execution timeout")
	 }
```